docs/DeveloperGuide.md

```diff
@@ -310,38 +310,41 @@ one priority only.
 
 #### Proposed Implementation
 
-The priority of a bug will be represented by `Priority` in the `seedu.address.model.bug` package. The `Priority` would
-be quite similar to other classes in the same package such as `Description`, `Name`, that is:
-- `Priority` will also have its own `MESSAGE_CONSTRAINTS` and `VALIDATION_REGEX`.
-- We will need to update the `Bug` to include an instance of `Priority`, and also update its constructor.
-- We will need to update `MESSAGE_USAGE` of `AddCommand` and `EditCommand`.
-- We will also need to update `EditCommand.BugDescriptor`.
-- We will add one more prefix constant called `PREFIX_PRIORITY` into `CliSyntax`.
-- We will update the `AddCommandParser#parse` and `EditCommandParser#parse` accordingly to read the `pr/` argument.
-- We will have to create `ParserUtil#parsePriority`.
-- We will need to include an instance of `Priority` into `seedu.address.storage.JsonAdaptedBug`, and update all
-of its methods to deal with priority.
-- We will need to update `seedu.address.ui.BugCard` and `BugListCard.fxml` to include priority.
-
 The followings are notable differences between `Priority` and other fields of `Bug`:
 - The `VALIDATION_REGEX` of `PRIORITY` will be set such that its constructor can only accept the `String` `"low"`, 
 `"medium"`, `"higher"` (either uppercase or lowercase)
-- Aside from the 3 possible states of `Priority` (`"low"`, `"medium"` or `"high"`), it will also have the state `"null"` 
+- Aside from the 3 possible states of `Priority` (`"low"`, `"medium"` or `"high"`), it will also have the state `""` 
 that represent when the bug have no priority indicated. This type of `Priority` will be create using an overload 
-version of the constructor that accept no argument (`new Priority("low")` will create a low priority, but 
-`new Priority()` will create a "null" priority. We will check if the `Priority` is "null" before showing it on the UI.
+version of the constructor that accept no argument: `new Priority("low")` will create a low priority, but 
+`new Priority()` will create an "empty" priority. 
+- We will check if the `Priority` is "empty" before showing it on the UI.
```

Comment:
Do you think we should structure it as steps to make it more consistent with other sections?

Reply:
Yes, I have added a step-by-step example.